### DISCUSSION TITLE
Merge pull request for GetMaterialApp to include restorationScopeId as parameter

### DISCUSSION THREAD

**Discussion by ac-arconomy** [jonataslaw/getx#2335] [General]
Hi Jonny,

This is a great project and seems to have a bit of a following - are you still actively maintaining it?

Please can you resolve the conflicts with merging the RestorationScopeId update.

https://github.com/jonataslaw/getx/pull/1851

Many Thanks
ac

**Comment by ac-arconomy** [comment]
@jonataslaw

**Comment by jonataslaw** [comment]
I'm actively maintaining it, and this PR is in conflict with the master branch. 
Do you have tested the beta version?
